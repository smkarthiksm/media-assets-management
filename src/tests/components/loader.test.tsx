import '@testing-library/jest-dom';
import { renderWithProviders } from '../test-utility';
import LoaderComponent from '../../components/loader/loader';
import { screen } from '@testing-library/react';

describe('LoaderComponent', () => {
  test('should render', () => {
    const { container } = renderWithProviders(<LoaderComponent />);
    expect(container).toMatchSnapshot();
  });

  test('should show loader', () => {
    renderWithProviders(<LoaderComponent />, {
      preloadedState: { loader: { isLoaderVisible: true } },
    });

    const loader = screen.getByRole('progressbar');
    expect(loader).toBeInTheDocument();
  });
});
