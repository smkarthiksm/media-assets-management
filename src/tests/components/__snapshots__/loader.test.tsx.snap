// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoaderComponent should render 1`] = `<div />`;
